### PR TITLE
Integrate ble home screen

### LINKED ISSUE #14: [TODO][libcen-and-ux] Bring back code commented out because of Firebase Quota
Remember to do this

### LINKED ISSUE #15: [TODO][libcen-and-ux] Bring back initBluetooth
Commenting out for emualtors

### LINKED ISSUE #56: [UI/UX][BLE] Integrate BLE with new homepage

### PR DESCRIPTION
**Pull request by nkumarcc**
- Main Fragment => Home Fragment
- Drop User Profile and Contact Events fragments (deprecated from Tags)
- Made `SharedViewModel` and `HomeViewModel`, shifted some logic to them
- Integrated BLE with home screen

Experimented with Firebase, found an issue that periodic downloads aren't working

### CODE REVIEW

File: app/src/main/java/org/covidwatch/android/ui/shared/SharedViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        .contactEventDAO()
+        .pagedAllSortedByDescTimestamp
+        .toLiveData(pageSize = 50)
+
+    private val _hasPossiblyInteractedWithInfected: LiveData<Boolean> = Transformations.map(contactEvents) {
+            cenList -> cenList.fold(false) {
+                isInfected: Boolean, cen: ContactEvent -> isInfected || cen.wasPotentiallyInfectious
+            }
+    }
+    val hasPossiblyInteractedWithInfected: LiveData<Boolean> get() = _hasPossiblyInteractedWithInfected
```
Comment (nkumarcc) [inline app/src/main/java/org/covidwatch/android/ui/shared/SharedViewModel.kt:27]: @madim Is this not wiring with `LiveData` correctly?

Comment (madim) [reply app/src/main/java/org/covidwatch/android/ui/shared/SharedViewModel.kt:27]: lgtm

Comment (nkumarcc) [reply app/src/main/java/org/covidwatch/android/ui/shared/SharedViewModel.kt:27]: Aaaayyy learning Android

File: app/src/main/java/org/covidwatch/android/ui/shared/SharedViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+class SharedViewModel(
+    private val isCurrentSickSharedPreferenceLiveData: IsCurrentSickSharedPreferenceLiveData,
+    application: Application
+) : AndroidViewModel(application) {
+
+    private val contactEvents =  CovidWatchDatabase.getInstance(application)
+        .contactEventDAO()
+        .pagedAllSortedByDescTimestamp
+        .toLiveData(pageSize = 50)
```
Comment (madim) [inline app/src/main/java/org/covidwatch/android/ui/shared/SharedViewModel.kt:20]: nit: this is what repository is for

**Comment by BurningAXE** [comment]
Are we going to drop this one in favour of @madim's one?

**Comment by madim** [comment]
> Are we going to drop this one in favour of @madim's one?

let's keep this PR for now, we'll see what to do after merging my [PR](https://github.com/covid19risk/covidwatch-android/pull/65)

**Comment by Apisov** [comment]
I'm closing it because we merged #65 in the end.

**Pull request closed without merging**